### ISSUE TITLE
Sending Emails - To Line Wrapping

### ISSUE THREAD

**Issue by jonathanchan** [mimestream/new-bugs#25]
**Describe the bug**
When adding a sizable number of email addresses in the To line of a new email, the To line does not word wrap. I have to use the cursor keys to scroll 

**To Reproduce**
1. Start a new email message
2. Add 10-20 email addresses to the To line
3. Verify that you're not able to see all the email addresses without manipulating the cursor.

**Expected behavior**
List of email addresses on the To line should be word wrap.

**Screenshots**
Please add screenshots to help explain your problem.

**Configuration:**
 - OS: macOS 10.15.6
 - Mimestream Version 0.6.26 (100)
![Markup 2020-10-12 at 14 46 07](https://user-images.githubusercontent.com/561437/95793153-d485cc00-0c99-11eb-98cd-1603899ccc5e.png)

**Comment by neiljhaveri** [comment]
Yeah, these fields need to expand up to maybe 3-4 lines and then scroll vertically
